core-src/annotatemain.cc

```diff
@@ -692,6 +699,12 @@ int main(int argc, char **argv)
               do_error("The --seed argument requires an integer parameter!");
             ++arg_index;
             srand((unsigned int)atoi(argv[arg_index]));
+        } else if (arg == "--load") {
+            if (arg_index+1 >= argc)
+              do_error("The -- load argument requires a valid file transcript to read!");
```

Comment:
Typo here (`-- load` for `--load`).
Also, isn't this `./annotate --load foo` just a synonym for `cat foo <(echo verbose) - | ./annotate --auto`? If so, I don't see a need for it.